current.go

```diff
@@ -20,6 +21,10 @@ func (r *VaultRole) Exists(config ExistsConfig) (bool, error) {
 		return false, nil
 	}
 
+	return roleExists(secret, config)
```

Comment:
I would like to handle the return values explicit to get the error masking. It is then easier to immediately see which stack the error went a long in case shit hits the fan. Also always useful for debugging.